src/scaffoldmaker/annotation/body_terms.py

```diff
@@ -0,0 +1,30 @@
+"""
+Common resource for body annotation terms.
+"""
+
+# convention: preferred name, preferred id, followed by any other ids and alternative names
+body_terms = [
+    ( "abdomen", "UBERON:0000916", "ILX:0725977" ),
+    ( "thorax", "ILX:0742178" ),
+    ( "neck", "UBERON:0000974", "ILX:0733967" ),
+    ( "head", "UBERON:0000033", "ILX:0104909" ),
```

Comment:
Groups head and neck don't really have the correct meaning. Perhaps these should include core and non-core in those regions, if they are needed?